src/RuleSet/Sets/PERCS10Set.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\RuleSet\Sets;
+
+use PhpCsFixer\RuleSet\AbstractRuleSetDescription;
+
+/**
+ * @internal
+ *
+ * PER Coding Style v1.0.0.
+ *
+ * @link https://github.com/php-fig/per-coding-style/blob/1.0.0/spec.md
+ */
+final class PERCS10Set extends AbstractRuleSetDescription
+{
+    public function getRules(): array
+    {
+        return [
+            '@PSR12' => true,
+        ];
+    }
+
+    public function getDescription(): string
+    {
+        return 'Rules that follow `PER Coding Style 1.0 <https://www.php-fig.org/per/coding-style/>`_.';
```

Comment:
Not sure this method is always called in a reStructuredText context. But since this is quite legible, maybe that's ok anyway.

Reply:
No idea.  That's how the existing `PERSet` was written, so I just copied-and-modified it.  Happy to change it to whatever.

Reply:
Right, let's keep it.